### PR TITLE
feat: Implemented filtering by file name and job name

### LINKED ISSUE #557: [MAJOR] No filtering by File name
No filtering by document name in the dashboard if they need to return to a specific document.

### PR DESCRIPTION
**Pull request by Tokhirov-Abzal**
Closes #557 and #555

### CODE REVIEW

File: web/src/connectors/tasks-table-connector/use-job-filter.ts [hunk 1]
```diff
@@ -18,29 +17,30 @@ export const useJobFilter = ({ fieldName }: Props) => {
...
     );
 
-    const renderJobFilter = useColumnPickerFilter<Job, string, unknown, string>(
+    const renderJobFilter = useColumnPickerFilter<string, string, unknown, string>(
         jobNames,
         fieldName,
         {
             showSearch: true,
-            getName: (item) => (typeof item === 'boolean' ? String(item) : item.name)
+            getName: (item) => (typeof item === 'boolean' ? String(item) : item)
```
Comment (sbilevich) [inline web/src/connectors/tasks-table-connector/use-job-filter.ts:43]: I believe we don't need getName here, as we have only names. Please try without it

File: web/src/connectors/tasks-table-connector/use-name-filter.ts [hunk 2]
```diff
@@ -18,30 +17,30 @@ export const useNameFilter = ({ fieldName }: Props) => {
...
     );
 
-    const renderNameFilter = useColumnPickerFilter<FileDocument, string, unknown, string>(
+    const renderNameFilter = useColumnPickerFilter<string, string, unknown, string>(
         documentNames,
         fieldName,
         {
             showSearch: true,
-            getName: (item) => (typeof item === 'boolean' ? String(item) : item.original_name)
+            getName: (item) => (typeof item === 'boolean' ? String(item) : item)
```
Comment (sbilevich) [inline web/src/connectors/tasks-table-connector/use-name-filter.ts:43]: The same

**Review by sbilevich — APPROVED** [review]

File: web/src/connectors/tasks-table-connector/use-job-filter.ts (hunk 1 above)
Comment (Tokhirov-Abzal) [reply web/src/connectors/tasks-table-connector/use-job-filter.ts]: Thanks for pointing it out. Removed

File: web/src/connectors/tasks-table-connector/use-name-filter.ts (hunk 2 above)
Comment (Tokhirov-Abzal) [reply web/src/connectors/tasks-table-connector/use-name-filter.ts]: Removed

**Pull request merged**